js/navigation.js

```diff
@@ -65,10 +65,10 @@
 		var self = this;
 
 		// Move up through the ancestors of the current link until we hit .nav-menu.
-		while ( -1 === self.className.indexOf( 'nav-menu' ) ) {
+		while ( self.classList.contains( 'nav-menu' ) ) {
 			// On li elements toggle the class .focus.
 			if ( 'li' === self.tagName.toLowerCase() ) {
-				if ( -1 !== self.className.indexOf( 'focus' ) ) {
+				if ( -1 !== self.classList.contains( 'focus' ) ) {
```

Comment:
We don't have to test equality since we are using now `contains`.

```suggestion
				if ( self.classList.contains( 'focus' ) ) {
```